test/models/host_facets/reported_data_facet_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+class ReportedDataFacetTest < ActiveSupport::TestCase
+  describe '#uptime_seconds' do
+    test 'should return uptime in seconds' do
+      host = FactoryBot.create(:host)
+      boot_time = 1.day.ago
+      host.create_reported_data(:boot_time => boot_time)
+      now = Time.zone.now.to_i
+      assert_equal host.uptime_seconds, now - boot_time.to_i
```

Comment:
Should this be inside `freeze_time` block to make sure we don't have some random failures because clocks are weird?

Reply:
Agreed, changed :)